ui-framework-documentation.md

```diff
@@ -113,100 +177,42 @@ sub run {
 }
 ```
 
+Or if we are using [external test data](https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/declarative-schedule-doc.md#test_data) (recommended)
```

Comment:
s/Or if we/In case we/

Reply:
It would be "In the case where we" to be proper english I think, though I'm not native either.

Reply:
yes, any of those would work.

Reply:
Here also, I think it's kinda too heavy sentence, that's why I chose "Or if we".